app/utils/issue_parser.py

```
@@ -0,0 +1,93 @@
+            if start < 1 or end < 1:
+                raise ValueError("Issue numbers must be positive")
+
+            if start > end:
+                raise ValueError(f"Range start ({start}) cannot exceed end ({end})")
+
+            result.extend(str(i) for i in range(start, end + 1))
+        else:
+            try:
+                issue_num = int(part)
+                if issue_num < 1:
+                    raise ValueError("Issue numbers must be positive")
```

Ruff (0.15.2)

[warning] Abstract `raise` to an inner function

(TRY301)